### PR TITLE
Fixed errors and descriptions

### PR DESCRIPTION
**Pull request by kyscg**

### CODE REVIEW

**Comment by kyscg** [comment]
@dynamicwebpaige May I have a review?

File: docs/datasets.md [hunk 1]
```diff
@@ -333,7 +333,8 @@ VALIDATION |     19,867
 High-quality version of the CELEBA
 dataset, consisting of 30000 images in 1024 x 1024 resolution.
 
-WARNING: this dataset currently requires you to prepare images on your own.
+WARNING: This dataset currently requires you to prepare images on your own. Tensorflow also provides an image generator
```
Comment (cyfra) [inline docs/datasets.md]: The datasets.md file is generated automatically using scripts/document_datasets.py and information from the datasets.

Please change the dataset descriptions in corresponding files instead.

Comment (kyscg) [reply docs/datasets.md]: Thanks for the feedback, I've made the changes in #130

**Comment by cyfra** [comment]
#130 was merged.

I'll close this one.

**Pull request closed without merging**